parts/windows/kuberneteswindowssetup.ps1

```diff
@@ -287,34 +286,33 @@ try
     icacls.exe "c:\k"
     Get-ProvisioningScripts
     Get-LogCollectionScripts
-
+    
     Write-KubeClusterConfig -MasterIP $MasterIP -KubeDnsServiceIp $KubeDnsServiceIp
-
-    Write-Log "Download kubelet binaries and unzip"
+    
     Get-KubePackage -KubeBinariesSASURL $global:KubeBinariesPackageSASURL
-
+    
     # This overwrites the binaries that are downloaded from the custom packge with binaries.
     # The custom package has a few files that are necessary for future steps (nssm.exe)
     # this is a temporary work around to get the binaries until we depreciate
     # custom package and nssm.exe as defined in aks-engine#3851.
     if ($global:WindowsKubeBinariesURL){
-        Write-Log "Overwriting kube node binaries from $global:WindowsKubeBinariesURL"
         Get-KubeBinaries -KubeBinariesURL $global:WindowsKubeBinariesURL
     }
 
+    
     Write-Log "Installing ContainerD"

```

Comment:
Remove this since you have added it in Install-Containerd-Based-On-Kubernetes-Version?